060_Distributed_Search/20_Scan_and_scroll.asciidoc

```diff
@@ -17,53 +14,46 @@ are made to the index after the initial search request has been made. It does
 this by keeping the old data files around, so that it can preserve its ``view''
 on what the index looked like at the time it started.
 
---
-
-`scan`::
-
 The costly part of deep pagination is the global sorting of results, but if we
 disable sorting, then we can return all documents quite cheaply. To do this, we
-use the `scan` search type.((("scan search type"))) Scan instructs Elasticsearch to do no sorting, but
-to just return the next batch of results from every shard that still has
-results to return.
+sort by `_doc`. This instructs Elasticsearch just return the next batch of 
+results from every shard that still has results to return.
 
-To use _scan-and-scroll_, we execute a search((("scan-and-scroll"))) request setting `search_type` to((("search_type", "scan and scroll")))
-`scan`, and passing a `scroll` parameter telling Elasticsearch how long it
-should keep the scroll open:
+To scroll through results, we execute a search request and set the `scroll` value to
+the length of time we want to keep the scroll open.  
 
 [source,js]
 --------------------------------------------------
-GET /old_index/_search?search_type=scan&scroll=1m <1>
+GET /old_index/_search?scroll=1m <1>
 {
     "query": { "match_all": {}},
+    "sort" : ["_doc"], <2>
     "size":  1000
 }
 --------------------------------------------------
 <1> Keep the scroll open for 1 minute.
+<2> `_doc` is the most efficient sort order. 
 
-The response to this request doesn't include any hits, but does include a
+The response to this request includes a
 `_scroll_id`, which is a long Base-64 encoded((("scroll_id"))) string. Now we can pass the
-`_scroll_id` to the `_search/scroll` endpoint to retrieve the first batch of
+`_scroll_id` to the `_search/scroll` endpoint to retrieve the next batch of
 results:
 
 [source,js]
 --------------------------------------------------
-GET /_search/scroll?scroll=1m <1>
-c2Nhbjs1OzExODpRNV9aY1VyUVM4U0NMd2pjWlJ3YWlBOzExOTpRNV9aY1VyUVM4U0 <2>
-NMd2pjWlJ3YWlBOzExNjpRNV9aY1VyUVM4U0NMd2pjWlJ3YWlBOzExNzpRNV9aY1Vy
-UVM4U0NMd2pjWlJ3YWlBOzEyMDpRNV9aY1VyUVM4U0NMd2pjWlJ3YWlBOzE7dG90YW
-xfaGl0czoxOw==
+GET /_search/scroll
+{
+    "scroll": "1m",
+    "scroll_id" : "cXVlcnlUaGVuRmV0Y2g7NTsxMDk5NDpkUmpiR2FjOFNhNnlCM1ZDMWpWYnRROzEwOTk1OmRSamJHYWM4U2E2eUIzVkMxalZidFE7MTA5OTM6ZFJqYkdhYzhTYTZ5QjNWQzFqVmJ0UTsxMTE5MDpBVUtwN2lxc1FLZV8yRGVjWlI2QUVBOzEwOTk2OmRSamJHYWM4U2E2eUIzVkMxalZidFE7MDs="
+}
 --------------------------------------------------
-<1> Keep the scroll open for another minute.
-<2> The `_scroll_id` can be passed in the body, in the URL, or as a
-    query parameter.
 
-Note that we again specify `?scroll=1m`.  The scroll expiry time is refreshed
+Note that we again set the scroll expiration to 1m.  The scroll expiry time is refreshed
```

Comment:
And here we can just truncate to "Note that we again set the scroll expiration to 1m."